test/sequential/sequencerStaking.sequencerJoining.test.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ *
+ * @group L1RolldownUpdates
+ */
+
+import { SequencerStaking } from "../../utils/rollDown/SequencerStaking";
+import { initApi } from "../../utils/api";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { User } from "../../utils/User";
+import { Sudo } from "../../utils/sudo";
+import { Assets } from "../../utils/Assets";
+import { GASP_ASSET_ID } from "../../utils/Constants";
+import { waitSudoOperationSuccess } from "../../utils/eventListeners";
+import { BN } from "@polkadot/util";
+
+let chain: any;
+let testUser: User;
+let stakeAmount: BN;
+
+beforeAll(async () => {
+  await initApi();
+  await setupApi();
+  stakeAmount = (await SequencerStaking.minimalStakeAmount()).addn(1000);
+});
+
+beforeEach(async () => {
+  //There shouldn't be any sequencer in activeSequencers
+  [testUser] = setupUsers();
+  chain = "Ethereum";
+  await Sudo.batchAsSudoFinalized(Assets.mintNative(testUser));
+  testUser.addAsset(GASP_ASSET_ID);
+});
+
+it("GIVEN User provides a stake by using StakeOnly action THEN User is not a sequencer", async () => {
+  const events = await Sudo.asSudoFinalized(
+    Sudo.sudoAsWithAddressString(
+      testUser.keyRingPair.address,
+      await SequencerStaking.provideSequencerStaking(
+        stakeAmount,
+        chain,
+        "StakeOnly",
+      ),
+    ),
+  );
+  const eventJoining = events.filter(
+    (x) => x.method === "SequencerJoinedActiveSet",
+  );
+  const eventReserved = events.filter((x) => x.method === "Reserved");
+  expect(eventJoining[0]).toBeUndefined();
+  expect(eventReserved[0].event.data[1].toHuman()).toContain(
+    testUser.keyRingPair.address,
+  );
+  expect(eventReserved[0].event.data[2]).bnEqual(stakeAmount);
+  const sequencersList = await SequencerStaking.activeSequencers();
+  expect(sequencersList.toHuman().Ethereum).not.toContain(
+    testUser.keyRingPair.address,
+  );
+});
+
+it("GIVEN User provides a stake by using StakeAndJoinActiveSet action THEN User is a sequencer", async () => {
```

Comment:
what if using StakeAndJoinActiveSet but stake < minimum? could you please add this test?

Reply:
Added